src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const res = [];
+  const status = { ...state };
+
+  for (let i = 0; i < actions.length; i++) {
+    switch (actions[i].type) {
+      case 'addProperties' :
```

Comment:
Try to move all cases to enum. Example of usage:

`case ActionTypes.AddProperty:`

Reply:
Thanks for comment.
I cant understand what you meen.

Reply:
Okey, i understand, but cant now fix.

Reply:
![image](https://github.com/mate-academy/js_stateful-clones/assets/78905554/05651ab3-a82a-43e3-a007-662298363851)

Reply:
This task only clone, i cant make how you want.

Reply:
```
const ActionTypes = {
  AddProperties: 'addProperties',
  ...
}

switch (actions[i].type) {
      case ActionTypes.AddProperties' :
...
```